prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -959,103 +959,6 @@ fi_addr_t rxr_pkt_insert_addr(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry
 	return rdm_addr;
 }
 
-/**
- * @brief process a received packet
- *
- * @param[in]	ep		endpoint
- * @param[in]	pkt_entry	received packet entry
- */
-void rxr_pkt_proc_received(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
```

Comment:
In this revision I wanted to mock this function, which is not possible because it is called in the same file. Maybe with your refactor I do not need the mock anymore.

Reply:
Thank you!

It would be ideal that we do not need to mock this function.

Mocking it will make testing it harder.